internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -256,20 +256,8 @@ func instanceServerCreateRun(ctx context.Context, argsI interface{}) (i interfac
 	// More format details in buildVolumeTemplate function.
 	//
 	if len(args.AdditionalVolumes) > 0 || args.RootVolume != "" {
-		// Get default project ID.
-		projectID := args.ProjectID
-		if projectID == nil {
-			projectID = args.OrganizationID
```

Comment:
`args.OrganizationID` and `args.ProjectID` MUST be propagated to buildVolumes or it will break he following syntax:

<img width="568" alt="image" src="https://user-images.githubusercontent.com/10092554/92584792-5e560a00-f294-11ea-92b7-97351e61cda5.png">